### PR TITLE
Vic/list view in global

### PR DESCRIPTION
**Pull request by vsauceda-cfa**
Here is the list view in the global search. seems to work. Definitely a workaround solution but it works. Would love to hear your thoughts on this one. I'm sure this solution can be improved. Also thinking about having a conditional for list and map view.

### CODE REVIEW

File: components/TagPane.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { SetStateAction, Dispatch, Fragment } from 'react'
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:1]: `SetStateAction` and `Dispatch` do not appear to be used in this component, so I don't think they need to be imported.

File: components/TagPane.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import { useRouter } from 'next/router'
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:2]: same as above with `useRouter`

File: components/TagPane.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import { FetchedDataSearch, LeafLoader, TagRecordCard } from './'
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:4]: same with `FetchedDataSearch`

File: components/TagPane.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import { Details, Paragraph } from '../ui'
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:5]: `Details` isn't used here, but I think it should be. I think that we should continue the same pattern of having 'foldable' content that we have used on other pages.

Also, without using `Details` for the list, it doesn't look like the map is rendering on mobile views

File: components/TagPane.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+import { TranslatedRecordResponse, OrgRecord } from '../types/records'
+import { ENGLISH } from '../constants/language'
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:8]: also unused

File: components/TagPane.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import { useGlobalSearch, useLanguage } from '../hooks'
+import styles from './RecordPane.module.css'
+
+export interface TagPaneProps {
+  orgInfo?: TranslatedRecordResponse
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:13]: should `orgInfo` be a required prop?

File: components/TagPane.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  setRecords?: Dispatch<SetStateAction<TranslatedRecordResponse>>
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:14]: this is unused in the component, so I don't think we need it in the props `interface`

File: components/TagPane.tsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
+import { SetStateAction, Dispatch, Fragment } from 'react'
+import { useRouter } from 'next/router'
+
+import { FetchedDataSearch, LeafLoader, TagRecordCard } from './'
+import { Details, Paragraph } from '../ui'
+
+import { TranslatedRecordResponse, OrgRecord } from '../types/records'
+import { ENGLISH } from '../constants/language'
+import { useGlobalSearch, useLanguage } from '../hooks'
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:9]: I don't think either of these is actually used.

File: components/TagPane.tsx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+  const { push, route } = useRouter()
+  const { language } = useLanguage()
+  const { searchResults, setSearchResults } = useGlobalSearch()
+
+  const url: string = `/search?query=${searchResults}`
+
+  const pushToCategory = () => {
+    if (url !== route) push(url, url)
+  }
+
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:27]: From what I can tell, none of this is used or necessary

File: components/TagPane.tsx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+import { ENGLISH } from '../constants/language'
+import { useGlobalSearch, useLanguage } from '../hooks'
+import styles from './RecordPane.module.css'
+
+export interface TagPaneProps {
+  orgInfo?: TranslatedRecordResponse
+  setRecords?: Dispatch<SetStateAction<TranslatedRecordResponse>>
+}
+
+const TagPane = ({ orgInfo, setRecords }: TagPaneProps) => {
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:17]: as `setRecords` is unused there is no need to deconstruct it.

File: components/TagPane.tsx [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+  if (!orgInfo) return <LeafLoader />
+
+  const recordsReady: boolean = Boolean(orgInfo?.records?.length)
+
+  return (
+    <div className={styles.RecordPane} role="menu">
+      {recordsReady &&
+        orgInfo.records.map((record: OrgRecord, i: number) => (
+          <Fragment key={i}>
+            <TagRecordCard record={record} url={url} />
```
Comment (timmalstead-cfa) [inline components/TagPane.tsx:37]: `url` is unused in `TagRecordCard`, so it is safe to remove it as a prop.

**Comment by timmalstead-cfa** [comment]
@vsauceda-cfa I have some comments on your latest pr. good work getting it working!

File: components/index.ts [hunk 12]
```diff
@@ -10,6 +10,7 @@ export { default as Disclaimer } from './Disclaimer'
 export { default as DisplayMap } from './DisplayMap'
 export { default as DiversionQualify } from './DiversionQualify'
 export { default as FetchedDataSearch } from './FetchedDataSearch'
+export { default as TagRecordCard } from './TagRecordCard'
```
[the remark this answers is not in the document]
Comment (vsauceda-cfa) [reply components/index.ts:13]: Done. Added one forgot the other 😞

File: types/records.ts [hunk 13]
```diff
@@ -4,7 +4,7 @@ export type Holder = { [title: string]: Array<string | number> }
 
 export interface TranslatedRecordResponse {
   offset?: string
-  category: string
+  category?: string
```
[the remark this answers is not in the document]
Comment (vsauceda-cfa) [reply types/records.ts:7]: No reason at all. I think I was playing around with it and forgot to take the ? mark off

File: pages/search/index.tsx [hunk 14]
```diff
@@ -1,25 +1,23 @@
 import { useState, useEffect } from 'react'
 import { useRouter } from 'next/router'
-
 import { POST, convertLocationsForMap } from '../../helpers'
 import { useGlobalSearch, useLanguage } from '../../hooks'
-
 import { LocationRecord, TranslatedRecordResponse } from '../../types/records'
+import TagPane from '../../components/TagPane'
 import DisplayMap from '../../components/DisplayMap'
```
[the remark this answers is not in the document]
Comment (vsauceda-cfa) [reply pages/search/index.tsx:7]: Thanks for the reminder....again

File: pages/search/index.tsx [hunk 15]
```diff
@@ -28,27 +26,24 @@ const GlobalSearchLanding = () => {
...
     }
     filterOrFetch()
   }, [searchResults])
-
   return (
     <>
+      {convertedLocRecords && <TagPane orgInfo={searchResults} />}
       {convertedLocRecords && <DisplayMap latLongInfo={convertedLocRecords} />}
     </>
   )
```
[the remark this answers is not in the document]
Comment (vsauceda-cfa) [reply pages/search/index.tsx:47]: Done and makes sense

File: components/TagRecordCard.tsx [hunk 16]
```diff
@@ -0,0 +1,69 @@
+import { Fragment, ReactElement } from 'react'
```
[the remark this answers is not in the document]
Comment (vsauceda-cfa) [reply components/TagRecordCard.tsx:1]: Ok. I switched up `OrgRecordCard` a little looks to be working fine so im removing `TagRecordaCard`

File: components/TagPane.tsx (hunk 1 above)
Comment (vsauceda-cfa) [reply components/TagPane.tsx:1]: Removed

File: components/TagPane.tsx (hunk 4 above)
Comment (vsauceda-cfa) [reply components/TagPane.tsx:5]: Ok added this in

File: components/TagPane.tsx (hunk 9 above)
Comment (vsauceda-cfa) [reply components/TagPane.tsx:27]: got it

**Comment by timmalstead-cfa** [comment]
@vsauceda-cfa this looks much cleaner. 


The only thing U'm wondering about is why you changed the props on `OrgRecordCard`?

Eliminating the props is causing some of the tests to break =>

Screen Shot 2021-05-19 at 5 20 03 PM

**Pull request merged**